scripts/file_resume.js

```diff
@@ -23,6 +23,42 @@ window.addEventListener("DOMContentLoaded", function() {
         return true;
     }
 
+    // polyfill for file.arrayBuffer()
+    function fileToArrayBuffer(file) {
+        if ('arrayBuffer' in file) {
+            return file.arrayBuffer();
+        }
+        return new Promise (function (resolve, reject) {
+            const reader = new FileReader();
+            reader.addEventListener("loadend", function() {
+                resolve(reader.result);
+            });
+            reader.addEventListener("error", function() {
+                reject();
+            });
+            reader.readAsArrayBuffer(file);
+        });
+    }
+
+    // polyfill for String.padStart(2, "0")
```

Comment:
do we need a polyfil? The support looks greater than our supported browser matrix:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart
https://www.pgdp.net/wiki/DP_Official_Documentation:General/Supported_Browsers

Reply:
Looks like we would have to support Chrome 57 but we only do 54

Reply:
Ah, missed that. I'm fine with keeping this, but do we know why we support chrome 54 specifically? Was it the last versino supported on xp or something like that? It was released in 2016.

Reply:
I don't know. According to browserstack xp only supports chrome 49. xp supports firefox 52 which is the reason for that and firefox 52 does PadStart.

Reply:
DP's supported browser list was last updated two-ish years ago in October of 2021. See [Supported Browsers](https://www.pgdp.net/wiki/DP_Official_Documentation:General/Supported_Browsers), and note the comment about Safari, Chrome and Opera on macOS: I couldn't find an official end-of-life schedule for macOS, but it looks like a three-year cycle. Big Sur (macOS 11) is no longer supported now.

How rigorous do we want to be in updating our supported browser list?

Reply:
To not boil the ocean, for this PR I think we should bump our minimum Chrome version to 57 and drop the polyfill. Based on the [platform support numbers for Chrome](https://en.wikipedia.org/wiki/Google_Chrome#Platforms) 52 -> 57 doesn't cross any threshold and seems very sensible.

Reply:
I'm happy to revisit all of our minimum browser versions if we need to, but it doesn't sound like this warrants that extra effort.